{
  "cve": "CVE-2016-9393",
  "mitre": {
    "cpes": [],
    "created": "2017-03-23T18:00:00+00:00",
    "description": "The jpc_pi_nextrpcl function in jpc_t2cod.c in JasPer before 1.900.17 allows remote attackers to cause a denial of service (assertion failure) via a crafted file.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2016/9xxx/CVE-2016-9393.json",
    "references": [
      "http://www.openwall.com/lists/oss-security/2016/11/17/1",
      "http://www.securityfocus.com/bid/94377",
      "https://access.redhat.com/errata/RHSA-2017:1208",
      "https://blogs.gentoo.org/ago/2016/11/16/jasper-multiple-assertion-failure",
      "https://bugzilla.redhat.com/show_bug.cgi?id=1396972",
      "https://github.com/mdadams/jasper/commit/f7038068550fba0e41e1d0c355787f1dcd5bf330",
      "https://usn.ubuntu.com/3693-1/"
    ],
    "title": null,
    "updated": "2024-08-06T02:50:37.791000+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "nvd": {
    "cpes": [
      "cpe:2.3:a:jasper_project:jasper:1.900.17:*:*:*:*:*:*:*"
    ],
    "created": "2017-03-23T18:59:00.817000+00:00",
    "description": "The jpc_pi_nextrpcl function in jpc_t2cod.c in JasPer before 1.900.17 allows remote attackers to cause a denial of service (assertion failure) via a crafted file.",
    "metrics": {
      "cvssV2_0": {
        "score": 4.3,
        "vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P"
      },
      "cvssV3_0": {
        "score": 5.5,
        "vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
      },
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2016/CVE-2016-9393.json",
    "references": [
      "http://www.openwall.com/lists/oss-security/2016/11/17/1",
      "http://www.securityfocus.com/bid/94377",
      "https://access.redhat.com/errata/RHSA-2017:1208",
      "https://blogs.gentoo.org/ago/2016/11/16/jasper-multiple-assertion-failure",
      "https://bugzilla.redhat.com/show_bug.cgi?id=1396972",
      "https://github.com/mdadams/jasper/commit/f7038068550fba0e41e1d0c355787f1dcd5bf330",
      "https://usn.ubuntu.com/3693-1/"
    ],
    "title": null,
    "updated": "2018-06-29T01:29:01.877000+00:00",
    "vendors": [
      "jasper_project",
      "jasper_project$PRODUCT$jasper"
    ],
    "weaknesses": [
      "NVD-CWE-Other"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [
        "cpe:/o:redhat:enterprise_linux:6",
        "cpe:/o:redhat:enterprise_linux:7",
        "cpe:2.3:a:jasper_project:jasper:1.900.17:*:*:*:*:*:*:*"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    },
    "created": {
      "data": "2016-10-28T00:00:00+00:00",
      "provider": "redhat"
    },
    "description": {
      "data": "The jpc_pi_nextrpcl function in jpc_t2cod.c in JasPer before 1.900.17 allows remote attackers to cause a denial of service (assertion failure) via a crafted file.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {
          "score": 4.3,
          "vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P"
        },
        "provider": "nvd"
      },
      "cvssV3_0": {
        "data": {
          "score": 5.5,
          "vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
        },
        "provider": "nvd"
      },
      "cvssV3_1": {
        "data": {},
        "provider": null
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": "Low",
        "provider": "redhat"
      }
    },
    "references": {
      "data": [
        "http://www.openwall.com/lists/oss-security/2016/11/17/1",
        "http://www.securityfocus.com/bid/94377",
        "https://access.redhat.com/errata/RHSA-2017:1208",
        "https://blogs.gentoo.org/ago/2016/11/16/jasper-multiple-assertion-failure",
        "https://bugzilla.redhat.com/show_bug.cgi?id=1396972",
        "https://github.com/mdadams/jasper/commit/f7038068550fba0e41e1d0c355787f1dcd5bf330",
        "https://nvd.nist.gov/vuln/detail/CVE-2016-9393",
        "https://usn.ubuntu.com/3693-1/",
        "https://www.cve.org/CVERecord?id=CVE-2016-9393"
      ],
      "providers": [
        "mitre",
        "nvd",
        "redhat"
      ]
    },
    "title": {
      "data": "jasper: insufficient SIZ marker segment data sanity checks",
      "provider": "nvd"
    },
    "updated": {
      "data": "2024-08-06T02:50:37.791000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [
        "jasper_project",
        "jasper_project$PRODUCT$jasper",
        "redhat",
        "redhat$PRODUCT$enterprise_linux"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    },
    "weaknesses": {
      "data": [
        "CWE-617",
        "NVD-CWE-Other"
      ],
      "providers": [
        "nvd",
        "redhat"
      ]
    }
  },
  "redhat": {
    "cpes": [
      "cpe:/o:redhat:enterprise_linux:6",
      "cpe:/o:redhat:enterprise_linux:7"
    ],
    "created": "2016-10-28T00:00:00+00:00",
    "description": "The jpc_pi_nextrpcl function in jpc_t2cod.c in JasPer before 1.900.17 allows remote attackers to cause a denial of service (assertion failure) via a crafted file.",
    "metrics": {
      "cvssV2_0": {
        "score": 4.3,
        "vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P"
      },
      "cvssV3_0": {
        "score": 5.5,
        "vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
      },
      "cvssV3_1": {},
      "threat_severity": "Low"
    },
    "redhat_repo_path": "2016/CVE-2016-9393.json",
    "references": [
      "https://nvd.nist.gov/vuln/detail/CVE-2016-9393",
      "https://www.cve.org/CVERecord?id=CVE-2016-9393"
    ],
    "title": "jasper: insufficient SIZ marker segment data sanity checks",
    "updated": "2016-10-28T00:00:00+00:00",
    "vendors": [
      "redhat",
      "redhat$PRODUCT$enterprise_linux"
    ],
    "weaknesses": [
      "CWE-617"
    ]
  }
}